versioned_docs/version-22.10/installation/installation-of-a-central-server/unattended.md

```diff
@@ -50,7 +50,7 @@ dnf update
 <TabItem value="Debian 11" label="Debian 11">
 
 ```shell
-apt update && apt upgrade
+apt update && apt install --only-upgrade
```

Comment:
--only-upgrade is used with an argument
so if you don't need to update a specific package, you don't need to replace "apt upgrade" by "apt install --only-upgrade"
I just tested it and if you use "apt update && apt install --only-upgrade" it will not update any package because e don't specified any argument